src/govuk/components/footer/footer.yaml

```diff
@@ -67,6 +67,32 @@ params:
       type: object
       required: false
       description: HTML attributes (for example data attributes) to add to the anchor in the footer navigation section.
+- name: contentLicence
+  type: object
+  required: false
+  description: Object containing options for the content licence.
+  params:
+  - name: text
+    type: string
+    required: false
+    description: If `html` is set, this is not required. Text to use within the disclosed part of the details element. If `html` is provided, the `text` option will be ignored. If neither are provided, the Open Government Licence is used.
```

Comment:
I wonder if this description implies that if you pass in your own text (or html), the Open Government Licence isn't used? Which isn't quite correct, because the logo will still be there.

Reply:
Perhaps. Most other examples of this kind include the actual text string that it falls back to, but in the case of the content licence this is fairly long and contains an embedded link, so including it wholesale didn't seem like a good idea.

Reply:
Maybe just a tweak needed then, e.g: `If neither are provided, the text for he Open Government Licence is used.`?